libcaf_core/caf/logger.hpp

```diff
@@ -64,8 +64,10 @@ class logger {
   struct event {
     event* next;
     event* prev;
+    int level;
+    std::string prefix;
```

Comment:
Where does this prefix come from? What is the purpose of this? I don't see it used anywhere. All it currently does is padding each single log entry with a whitespace.